internal/printer/printer_test.go

```diff
@@ -3425,6 +3425,18 @@ const items = ["Dog", "Cat", "Platipus"];
 				code: `${$$maybeRenderHead($$result)}<div data-astro-transition-persist="foo"${$$addAttribute($$renderTransition($$result, "peuy4xf7", "", "foo"), "data-astro-transition-scope")}></div>`,
 			},
 		},
+		{
+			name: "transition:reload is converted to a data attribute",
+			source: `
+			---
```

Comment:
transition:reload does not create a transition scope.
But the test code below checks for "transition:" in the 'source' and insists on that import when present. 
Maybe that "if" should be changed to 'if tt.transitions {' (or at least be guarded by tt.transitions)

Reply:
I agree, guarding with `tt.transitions` makes more sense